.github/workflows/vale.yml

```
@@ -11,7 +11,8 @@ on:
 jobs:
   vale:
     name: linter
-    runs-on: ubuntu-latest
+    # workaround for https://github.com/errata-ai/vale-action/issues/128
+    runs-on: ubuntu-24.04
```

I think you intended 22.04 since the Vale action is incompatible with 24.04.
```suggestion
    runs-on: ubuntu-22.04
```

---

I was trying this workaround - https://github.com/errata-ai/vale-action/issues/128#issuecomment-2402360989

name: reviewdog
on: [pull_request]

jobs:
  vale:
    name: runner / vale
    runs-on: ubuntu-24.04
    steps:
      - uses: actions/checkout@v3
      - run: |
          venv="$HOME/.local/share/venv"
          python3 -m venv "$venv"
          echo "$venv/bin" >> $GITHUB_PATH
      - uses: errata-ai/vale-action@reviewdog

---

But if it dos not work, will try 22.04

---

It actually worked! But now there is a new error - Asciidoctor not found. So I will try 22.04

---

In that case, you don't need to change the `ubuntu-latest` value since that resolves to 24.04 anyway and the workaround would also work on 22.04.